app/client-app/src/components/shared/SurveyPage.jsx

```diff
@@ -148,16 +159,28 @@ const SurveyPage = ({
               </Badge>
             )}
           </div>
-          <Button
-            size="lg"
-            disabled={!nextEnabled || isBusy}
-            isLoading={isBusy}
-            colorScheme={nextEnabled ? "blue" : "gray"}
-            onClick={handleNextClick}
-            rightIcon={!lastPage && <FaChevronRight />}
-          >
-            {lastPage ? "Finish" : "Next"}
-          </Button>
+          <Stack spacing={3} direction="row" align="center">
+            <Button
+              size="md"
```

Comment:
Might just be me but I having 2 different size buttons next to each other doesn't look right? I think the outline works well though, but could make this large as well

Reply:
![image](https://github.com/decsys/decsys/assets/90258816/ef581dff-f079-4260-be14-d5377a4a3a7b)
Would this be better?

Reply:
![image](https://github.com/decsys/decsys/assets/90258816/35873cf7-bec8-4879-97f4-82b0229f112c)
The button is medium intentionally as its a lesser important one, I've updated it so it has a better outline by increasing the width